apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -67,12 +67,16 @@ function removeBehavior(sprite, behavior) {
   sprite.behaviors.splice(index, 1);
 }
 
+function createBehavior(func, extraArgs) {
+  return {
+    func,
+    extraArgs,
+  };
+}
+
 function normalizeBehavior(behavior) {
   if (typeof behavior === 'function')  {
-    behavior = {
-      func: behavior,
-      extraArgs: [],
-    };
+    return createBehavior(behavior, []);
```

Comment:
Do you think we'll ever want instance methods on these behavior hashes?  If so, maybe prefer:

```js
function Behavior(func, extraArgs) {
  this.func = func;
  this.extraArgs = extraArgs;
}

// Usage
new Behavior(behavior, []);
```

Reply:
I'm not sure if we'll ever need instance methods, but I think I like that better anyway.